### ISSUE TITLE
[Improvement] Add option to hide units or whole categories

### ISSUE THREAD

**Issue by VibrantTreeStump** [ferraridamiano/ConverterNOW#151]
The Android app is amazing, it works so well.
It would be very nice if there was the option to hide individual units, to be able to customize the interface exactly to one's needs. The same would be nice for categories. 

To quickly show units that are hidden I suggest a small button to temporarily unhide all objects. This way you could hide units you normally don't need but still have them quickly available in case you ever need them.

**Comment by alekksander** [comment]
Yes please. Especially for currency (not able to change order too).

**Comment by ferraridamiano** [comment]
I'm starting to work on this
